docs/Makefile

```diff
@@ -118,7 +118,7 @@ pseudoxml:
 	@echo "Build finished. The pseudo-XML files are in $(BUILDDIR)/pseudoxml."
 
 preview:
-	../scripts/previewer $(BUILDDIR)/html
+	python ../scripts/previewer $(BUILDDIR)/html
```

Comment:
@waveform80 Why do the scripts in the `scripts` directory use `#!/usr/bin/python3` rather than `#!/usr/bin/env python3` ? (which I believe would make this change unnecessary)